index.html

```diff
@@ -34,8 +34,8 @@ <h1>Welcome to Liam.L's CSD120 Website!</h1>
             <a href= "images/brilliant_thumbnail.jpg">
             <img
             src= "images/brilliant_thumbnail_300.jpg" alt= "a brilliant photo of yours trully"
-            sizes= "(max-width:500px) 150px, (max-width:800px) 300px"
```

Comment:
Just leave 300px on its own at the end to make it the default size when no other media condition is met